be/src/formats/parquet/encoding_dict.h

```diff
@@ -130,8 +130,8 @@ class DictDecoder final : public Decoder {
         T* __restrict__ data = data_column->get_data().data() + cur_size;
 
         auto ret = _rle_batch_reader.GetBatchWithDict(_dict.data(), _dict.size(), data, count);
-        if (UNLIKELY(ret <= 0)) {
-            return Status::InternalError("DictDecoder GetBatchWithDict failed");
+        if (UNLIKELY(ret < 0)) {
```

Comment:
as i know, if this page has no value at all, we will just jump to next page based on page header.